### PR TITLE
ID23597: receive news field

### PR DESCRIPTION
**Pull request by sviridovartem**

### CODE REVIEW

File: webapplication/user/admin.py [hunk 1]
```diff
@@ -26,14 +26,15 @@ def save(self, commit=True):
 
 class UserAdmin(BaseUserAdmin):
     form = UserForm
-    list_display = ('username', 'email', 'is_staff', 'is_active', 'is_superuser', 'area_limit_ha')
+    list_display = ('username', 'email', 'is_staff', 'is_active', 'is_superuser', 'area_limit_ha', 'receive_news')
```
Comment (adolgarev) [inline webapplication/user/admin.py:29]: Add this field to filters

Comment (sviridovartem) [reply webapplication/user/admin.py:29]: done

File: webapplication/user/admin.py [hunk 2]
```diff
@@ -26,14 +26,16 @@ def save(self, commit=True):
...
+    list_filter = ('receive_news', )
```
Comment (adolgarev) [inline webapplication/user/admin.py:30]: This overrides the default filter https://github.com/django/django/blob/main/django/contrib/auth/admin.py#L77
Add at least ("is_staff", "is_active", "groups") too

Comment (sviridovartem) [reply webapplication/user/admin.py]: fixed

**Review by adolgarev — APPROVED** [review]

**Pull request merged**